en/firefox/browsers/mobile/focus.ftl

```diff
@@ -20,20 +20,36 @@ mobile-focus-send-a-download-link-to-your = Send a download link to your phone.
 mobile-focus-scan-the-qr-code-to-get-started = Scan the QR code to get started
 mobile-focus-scan-alt-text = Scan the QR code to get { -brand-name-firefox-focus }
 
+# Outdated string
 mobile-focus-disappear-your-history = Disappear your history
+
+mobile-focus-delete-your-history = Delete your history
+
 mobile-focus-easily-erase-your = Easily erase your history, passwords and cookies, so unwanted ads don’t follow you around online. Just tap the erase button on the search field and it’s gone.
 
 mobile-focus-take-private-mode = Take private mode to the next level
+
+# Outdated string
 mobile-focus-private-browsing-on = “Private browsing” on most browsers isn’t really all that private. { -brand-name-firefox-focus } is next-level, always on privacy and it’s backed by { -brand-name-mozilla }, the non-profit that fights for your rights on the web.
 
+mobile-focus-private-browsing-on-v2 = “Private browsing” on most browsers isn’t really all that private. { -brand-name-firefox-focus } is next-level, private by default and it’s backed by { -brand-name-mozilla }, the non-profit that fights for your rights on the web.
```

Comment:
This sentence seems missing a word, `{ -brand-name-firefox-focus } is next-level,`. Next level what??

Reply:
That's an americanism I guess. 

> Something or someone on a level above all the rest of the world. The person on the 'NEXT LEVEL' usually thinks differently from everyone, hence why he is known as NEXT LEVEL.
> 
> A thing can also be called NEXT LEVEL. Anything above average basically.

From https://www.urbandictionary.com/define.php?term=next%20level

Reply:
Thanks for the explanation. I understand `next level`. When it's hyphenated, I expect there is a word after this.

Reply:
good point. I think it being hyphenated here is a mistake.

Reply:
I think that may be deliberate, it's just very "west coast American." Maybe we need a simpler version for canonical English and if they really want to keep "next-level" we can do it in the en-US file. I'll ask.

Reply:
We're changing it to "{ -brand-name-firefox-focus } offers next-level privacy by default" which is MUCH clearer.

Reply:
Yeah, the comma threw things off. This is the proper expressions.